### PR TITLE
need to set last_signed_in for mobile iam users

### PR DESCRIPTION
**Pull request by jperk51**
## Description of change
Needed to set last_signed_in for mobile iam users in order to user EVSS services

## Original issue(s)
No issue created, hot fix

## Things to know about this PR
Set last_signed_in and updated tests to check that the field is non-null

### CODE REVIEW

File: spec/lib/iam_ssoe_oauth/session_manager_spec.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@
       before do
         VCR.use_cassette('iam_ssoe_oauth/introspect_active') do
           @user = session_manager.find_or_create_user
+          expect(@user.last_signed_in).not_to be_nil
```
Comment (kreek) [inline spec/lib/iam_ssoe_oauth/session_manager_spec.rb:22]: Could this go down into an `it` block? Might also want to check that it's a time in addition to not nil

Comment (jperk51) [reply spec/lib/iam_ssoe_oauth/session_manager_spec.rb:22]: will do

**Review by kreek — APPROVED** [review]
LGTM 🚀

**Review by thilton-oddball — APPROVED** [review]
LGTM

**Pull request merged**